tests/e2e/testsLibrary/LsTests.ts

```diff
@@ -47,42 +47,43 @@ const ide: Ide = e2eContainer.get(CLASSES.Ide);
         await editor.pressControlSpaceCombination(openedTab);
         await editor.waitSuggestion(openedTab, suggestionText);
     });
- }
+}
 
- export function autocomplete(openedTab: string, line: number, char: number, expectedText: string) {
+export function autocomplete(openedTab: string, line: number, char: number, expectedText: string) {
     test('Autocomplete', async () => {
         await editor.moveCursorToLineAndChar(openedTab, line, char);
         await editor.pressControlSpaceCombination(openedTab);
         await editor.waitSuggestionContainer();
         await editor.waitSuggestionWithScrolling(openedTab, expectedText);
     });
- }
+}
 
- export function waitLSInitialization(startingNote: string, startTimeout: number, buildWorkspaceTimeout: number) {
+export function waitLSInitialization(startingNote: string, startTimeout: number, buildWorkspaceTimeout: number) {
     test('LS initialization', async () => {
         await ide.checkLsInitializationStart(startingNote);
         await ide.waitStatusBarTextAbsence(startingNote, startTimeout);
         await ide.waitStatusBarTextAbsence('Building workspace', buildWorkspaceTimeout);
     });
- }
+}
 
- export function codeNavigation(openedFile: string, line: number, char: number, codeNavigationClassName: string) {
-    test('Codenavigation', async () => {
+export function codeNavigationGoTo(openedFile: string, line: number, char: number, codeNavigationClassName: string, keyCombination: string = Key.chord(Key.CONTROL, Key.F12)) {
```

Comment:
I'm not sure if `GoTo` is adding any value here - the `codeNavigation` already says that you should be navigated somewhere. Maybe it would be worth adding if it should redirect you to `implementation`, `definition`, `references`...

Reply:
I wanted to prepare it for the two approaches: `codeNavigationGoTo` and `codeNavigationPeek` as is explained in this issue: https://github.com/eclipse/che/issues/17995

Reply:
I see, so what about renaming this method to `codeNavigationGoToImplementation`?

Reply:
Not necessarily, because based on what key combination we send, we can go do Definition, Implementation, References, etc.

Reply:
GoTo - we expect it to open a new file
Peek - we expect it to show the in-editor peek window